src/index.js

```diff
@@ -88,9 +75,11 @@ const getFirstReplacementValueForNode = (obj, comparator, nodePath) => {
  * @return {undefined}
  */
 const processNode = (replacements, nodePath, replaceFn, comparator) => { // eslint-disable-line
-  const replacement = getFirstReplacementValueForNode(replacements, comparator, nodePath);
-  if (replacement) {
-    replaceAndEvaluateNode(replaceFn, nodePath, replacement);
+  const replacementKey = getSortedObjectPaths(replacements)
```

Comment:
Pretty much, I had to do this because there was no way of knowing whether `getFirstReplacementValueForNode` returned `null` because `replacements` didn't contain `replacementKey`, or because `replacements.get(replacementKey)` is set to `null`, or worse, it was set to `undefined` but the func returned `null` instead because of the `|| null`.

I could make `getFirstReplacementValueForNode` return `{replacement: null}`, `{replacement: 'abab'}`, `{replacement: undefined}`, etc. if the user actually defined a replacement, and `null` otherwise.  But I'd rather just update the docs to `processNode`.